File: Java/algoritmo3.java
import java.util.*;

public class algoritmo3 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a temperatura em Celsius: ");
        double temperaturaCelsius = scanner.nextDouble();
        double temperaturaFahrenheit = (temperaturaCelsius * 9 / 5) + 32;

        System.out.printf("%.2f graus Celsius é equivalente a %.2f graus Fahrenheit.%n", temperaturaCelsius, temperaturaFahrenheit);

        scanner.close();
    }
}

File: C/algoritmo3.c
#include <stdio.h>

int main() {
    double temperaturaCelsius, temperaturaFahrenheit;

    printf("Digite a temperatura em Celsius: ");
    scanf("%lf", &temperaturaCelsius);

    temperaturaFahrenheit = (temperaturaCelsius * 9 / 5) + 32;

    printf("%.2f graus Celsius é equivalente a %.2f graus Fahrenheit.\n", temperaturaCelsius, temperaturaFahrenheit);

    return 0;
}
